src/includes/getting-started-install/apple.macos.mdx

```diff
@@ -11,4 +11,6 @@ target 'YourApp' do
 end
 ```
 
-Afterwards run `pod install`.
+Then run `pod install`.
+
+We also support alternate [installation methods](install/).
```

Comment:
Whatever we agree about the comment on the apple.ios.mdx would go here too

Reply:
We'll keep it as is since that's our standard. When folks click that link, they'll see the list of install methods.